src/components/Footer/footerData/Info.js

```diff
@@ -0,0 +1,7 @@
+export const INFO = [
+  { id: 1, content: '권상현' },
+  { id: 2, content: '김유신' },
+  { id: 3, content: '박소윤' },
+  { id: 4, content: '유병문' },
+  { id: 4, content: '장연정' },
+];
```

Comment:
1. Footer 폴더 하위에 Footer.js, Footer.scss, FooterData.js 3가지 파일을 위치시키고
2. FooterData.js에 Info.js와 siteMap.js에 포함된 2개의 상수를 모두 포함시켜주세요. (아래 참고)
3. 아래 예시와 같이 하나의 파일에서 여러 상수를 export 시킬 수 있습니다!
4. import 부분은 해당 부분에 리뷰 남길게요!
 
```suggestion
export const INFO = [
  { id: 1, content: '권상현' },
  { id: 2, content: '김유신' },
  { id: 3, content: '박소윤' },
  { id: 4, content: '유병문' },
  { id: 4, content: '장연정' },
];

export const SITEMAP = [
  { id: 1, content: '이용약관' },
  { id: 2, content: '개인정보처리 방침' },
  { id: 3, content: '고객센터' },
  { id: 4, content: '운영정책' },
];
```

Reply:
감사합니다 바로 적용하겠습니다!